containers/payments/LossLoyaltyModal.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { Alert, ConfirmModal, ErrorButton, Paragraph } from 'react-components';
+
+const LossLoyaltyModal = ({ user = {}, ...rest }) => {
+    return (
+        <ConfirmModal
+            title={c('Title').t`Confirm loss of Proton loyalty benefits`}
+            confirm={<ErrorButton type="submit">{c('Action').t`Confirm`}</ErrorButton>}
+            {...rest}
+        >
+            <Paragraph>{c('Info').t`As a long-term paid user, you are entitled to Proton Loyalty Benefits.`}</Paragraph>
```

Comment:
`Proton loyalty benefits` (without the capitalization) to be consistent with the title of the modal?